README.md

```diff
@@ -1,24 +1,218 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+Not Available yet!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You should be having [Ruby](https://ruby-lang.org/en/) installed on your computer
+- Nodejs Installed on your computer
+-
+
+### Setup
+
+To get a local copy up and running follow these simple example steps.
+
+1. Navigate to a folder where you want the cloned file to appear
+
+2. Clone this repo in your terminal or git bash using the command
+```sh
+  cd hello-rails-backend
+  git clone https://github.com/degisew/hello-rails-backend.git
+```
+
+3. run the following command to install all the necessary dependencies including the linters
+
+ ```sh
+  cd hello-rails-backend
+  gem install
+``` 
+
+4. Navigate to the folder called hello-rails-backend/ and open this project using your editor
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  cd hello-rails-backend
+ rails server
+```
+- This will start the rails server in your browser
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  bin/ rails rsepc sepc/
+```
+### Deployment
+
+You can deploy this project using:
+
+- Heroku
+- Render.com
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Degisew**
+
+- GitHub: [@degisew](https://github.com/degisew)
+- Twitter: [@Dj_etiya](https://twitter.com/DJ_etiya)
+- LinkedIn: [Degisew](https://linkedin.com/in/degisew-mengist)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [Deploy Online]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/degisew/hello-rails-backend/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a ⭐️ to repo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank the Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/degisew/hello-rails-backend/blob/dev/LICENSE) licensed.
+
```

Comment:
- [ ] Kindly fix this link is broken and it's giving me a 404 error. This way the readme file will look professional without broken links.